lib/games_manager.rb

```diff
@@ -56,11 +54,8 @@ def percentage_ties
   def count_of_games_by_season
     games_in_season = Hash.new
     games.each do |game|
-      if games_in_season[game.season].nil?
-        games_in_season[game.season] = 1
-      else
-        games_in_season[game.season] += 1
-      end
+      games_in_season[game.season] =0 if games_in_season[game.season].nil?
```

Comment:
Very clean, pop a space between the = and 0

Reply:
agreed.